CONTRIBUTE.md

```diff
@@ -26,3 +26,21 @@ This is a community built document. Dive in and help out!
         ```sh
         (relevant branch)$ git blah blah
         ```
+
+## Translation
+
+We are using with a translation tool called [GitLocalize][gl] and follow the steps to get started with your contribution:
+
+1. Go to [GitLocalize's k88hudson/git-flight-rules repository][gl-repo].
+1. Sign up using your GitHub account :octocat:.
+1. Find the document you are going to translate. 
+1. Happy translating :sparkles: .
+1. When you are done, send the translation for reviews.
+1. Reviewed translation will be sent as a Pull Request to GitHub by language admin in the community.
```

Comment:
The reviewed translation will be sent as a Pull Request to GitHub by the language admin in the community.

Reply:
Sound good but i just replace `admin` with `moderator` which is used in GitLocalize